packages/fetchai/connections/p2p_libp2p_client/connection.py

```diff
@@ -174,7 +174,7 @@ async def connect(self) -> None:
             self._process_messages_task = asyncio.ensure_future(
                 self._process_messages(), loop=self._loop
             )
-        except (CancelledError, Exception) as e:
+        except (CancelledError, Exception) as e:  # pragma: no cover
```

Comment:
on my local machine, with file (missing from develop) `test_errors.py`, this is already covered by `tests/test_packages/test_connections/test_p2p_libp2p_client/test_errors.py::TestLibp2pClientConnectionFailureNodeNotConnected::test_node_not_running`
(https://github.com/fetchai/agents-aea/blob/feature/libp2p-improve-coverage/tests/test_packages/test_connections/test_p2p_libp2p_client/test_errors.py#L51)